src/integration/HubspotApiModel.ts

```diff
@@ -17,15 +15,21 @@ export class HubspotApiModel {
      * @param objectId objectID of contact to query
      * @param paramToGet query parameters to filter by. eg. 'email'.
      */
+
+    //#todo: suggestion renaming objectID to contactObjectID
     public async getContact(objectId: number, paramToGet?: string){
-        const headers = { accept: 'application/json' };
+        const headers = { accept: 'application/json',authorization:`Bearer ${this.hub_access_token}` };
         const url = `https://api.hubapi.com/crm/v3/objects/contacts/${encodeURIComponent(objectId)}`;
         let qs;
+        console.log("this is header",headers)
+        console.log("this is auth",this.hub_access_token)
         if (paramToGet){
-            qs = {"properties": 'email', "archived": 'false', "hapikey": `${this.HAPIKEY}`};
+            qs = {archived: 'false'}
+
+            //qs = {"properties": 'email', "archived": 'false'}
```

Comment:
make sure to remove commented code. Or put another comment why it should be there